src/components/RepoCard.tsx

```diff
@@ -0,0 +1,46 @@
+import '@styles/RepoCard.styles.scss'
+import { RoundedTag } from './RoundedTag'
+import { Col, Row, Typography } from 'antd'
+import { StarOutlined } from '@ant-design/icons'
+import { GithubRepoContent } from '@types'
+import { replaceShortcodeEmojis, transformRepoTopics } from '@utils'
+
+const { Title, Text } = Typography
+
+interface RepoCardProps {
+  repo: GithubRepoContent
+}
+export const RepoCard = ({ repo }: RepoCardProps) => {
+  const topics = transformRepoTopics(repo.topics, repo.language)
+
+  return (
+    <a href={repo.html_url} target='_blank' rel='noopener noreferrer' className='repo-card'>
+      <div className='card-bg'>
+        <Row justify='space-between' align='middle' className='card-row'>
+          <Col span={3}>{repo.language && <RoundedTag tagName={repo.language} />}</Col>
```

Comment:
if no repo.language, "false" will be rendered instead, which is unwanted, a ternary operator: 
```suggestion
          {repo.language ? 
                <Col span={3}>
                    <RoundedTag tagName={repo.language} />
                </Col>
                : null
          }
```

or a short-circuit 
```suggestion
          {repo.language && 
                <Col span={3}>
                    <RoundedTag tagName={repo.language} />
                </Col>
           }
```
to determine whether to show the whole Col or not in the first place would solve this problem

Reply:
since we already short-circuited other datapoints in this component, let's keep the consistency and use it instead of a ternary operator

Reply:
I did it for  `language` and  `stargazers_count` as well